backend/plumber.go

```diff
@@ -19,10 +19,9 @@ import (
 )
 
 const (
-	ifnamesiz    = 16
-	maxretries   = 5
-	retryttl     = time.Second * 5
-	peercheckttl = time.Second * 5
+	ifnamesiz  = 16
+	maxretries = 5
+	retryttl   = time.Second * 5
```

Comment:
The default one should be more. 5 second is too much looking at how often 1 clusters calls potato.